pkg/operator/operator.go

```diff
@@ -44,10 +47,41 @@ const (
 	cronInterval          = 5  // seconds
 )
 
-var markedWorkflows = strset.New()
+var (
+	awsClient       *aws.Client
+	telemtryClient  *telemetry.Client
+	markedWorkflows = strset.New()
+)
 
 func main() {
-	telemetry.ReportEvent("operator.init")
+	conf := config.NewFromEnv()
+	config.AWS = aws.New(conf.Region, conf.Bucket)
```

Comment:
Actually, now I'm wondering if we should use the lower-case `init()` just for `config`, to ensure it's always properly initialized? Also maybe we should use it for `argo` and `spark` since those are singletons which downstream packages (like `workflows`) rely on being initialized? we can discuss live